incubator/parse/templates/server-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-server
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        component: "{{ .Chart.Name }}-server"
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.parseServer.image }}"
+        imagePullPolicy: {{ default "" .Values.parseServer.imagePullPolicy | quote }}
+        env:
+        - name: PARSE_SERVER_HOST
+          value: "0.0.0.0"
+        - name: PARSE_SERVER_PORT
+          value: {{ .Values.parseServer.port | quote }}
+        - name: PARSE_SERVER_MOUNT_PATH
+          value: {{ .Values.parseServer.mountPath | quote }}
+        - name: PARSE_SERVER_APP_ID
+          value: {{ .Values.parseServer.appId | quote }}
+        - name: PARSE_SERVER_MASTER_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "fullname" . }}
+              key: master-key
+        - name: MONGODB_HOST
+          value: {{ template "mongodb.fullname" . }}
+        - name: MONGODB_PORT
+          value: "27017"
+        ports:
+        - name: server-http
+          containerPort: {{ .Values.parseServer.port }}
+        livenessProbe:
+          exec:
+            command:
```

Comment:
AFAIK Parse is a HTTP service. Could be better do an httpGet here too like here https://github.com/bitnami/charts/pull/208/files#diff-b5257a603d2f171b3d6dc63ea5724a49R48